code/modules/admin/callproc/callproc.dm

```diff
@@ -26,7 +26,7 @@ GLOBAL_PROTECT(AdminProcCallHandler)
 
 /mob/proccall_handler/Initialize(mapload)
 	. = ..()
-	if(GLOB.AdminProcCallHandler)
+	if(GLOB.AdminProcCallHandler && GLOB.AdminProcCallHandler != src)
```

Comment:
Can't you just do GLOB.AdminProcCallHandler != src? since if it's null, that statement would be false anyway

Reply:
if it is null, it will not equal src.

Reply:
I'll make it so that it sets GLOB.AdminProcCallHandler to itself if this check passes for redundancy

Reply:
Oh I see, totally understandable